maquetado/index.html

```diff
@@ -17,31 +17,31 @@ <h2 class="navbar-text">Developer Tools</h2>
   <h2 class="title">Front-End Tools</h2>
   <div class="container">
     <div class="wrap">
-      <div class="technology">
+      <div class="technology first-card">
         <img class="logo" src="./assets/react_icon.svg">
         <h4 class="technology-title">ReactJS</h4>
         <h4 class="technology-release">Release:2013</h4>
         <p class="technology-description">React is a declarative, component-based JavaScript library for building user interfaces.</p>
       </div>
-      <div class="technology">
+      <div class="technology cards">
```

Comment:
Instead of adding style to every tech, why do you think of changing the `technology` class and just override in the case of `first-card`?